native/src/navigation/OnboardingNavigator.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+import { createStackNavigator } from "@react-navigation/stack";
+import { OnboardingStackParamList } from "./types/types";
+import SendPhoneVerificationCode from "../feature/onboarding/screens/SendPhoneVerificationCode";
+import CheckCode from "../feature/onboarding/screens/CheckCode";
+
+const { Navigator, Screen } = createStackNavigator<OnboardingStackParamList>();
+
+const OnboardingNavigator = () => {
+  return (
+    <Navigator>
+      <Screen name="SendCode" component={SendPhoneVerificationCode} />
+      <Screen name="CheckCode" component={CheckCode} />
+    </Navigator>
+  );
+};
```

Comment:
Esto es correcto, pero hay un tip para hacerlo todavía más ligero: https://www.sitepoint.com/shorthand-javascript-techniques/#10implicitreturnshorthand

*10. Implicit Return Shorthand*

```suggestion
const OnboardingNavigator = () =>  (
    <Navigator>
      <Screen name="SendCode" component={SendPhoneVerificationCode} />
      <Screen name="CheckCode" component={CheckCode} />
    </Navigator>
  );
```

Reply:
- He atendido tu comentario, gracias por el tip y la lectura.